projboard/tests/test_user.py

```diff
@@ -1,54 +1,41 @@
 import pytest
 from projboard.models.user import User
-
-
-NAME = "RAWAD"
-NICKNAME = "USER3"
-EMAIL = "rawad@gmail.com"
-PASSWORD = "123456"
+NAME = "my_username"
+NICKNAME = "my_nickname"
+EMAIL = "my_email@gmail.com"
+PASSWORD = "my_password"
 NOT_EXISTED_USER = "undefined_user"
 
 
+@pytest.mark.django_db
 class TestUserModel:
 
-    @pytest.fixture
-    def generate_user(self):
-        user = User(email=EMAIL, password=PASSWORD, name=NAME, nickname=NICKNAME)
-        user.save()
-        return user
-
-    @pytest.mark.django_db
-    def test_generate_user(self, generate_user):
+    def test_create_user(self, user):
         # Test generarting new user in DB
-        assert generate_user in User.objects.all()
-
-    @pytest.mark.django_db
-    def test_create_user(self):
-        # Testing if generate_user
-        user = User.create_user(EMAIL, PASSWORD, NAME, NICKNAME)
         assert user in User.objects.all()
 
-    @pytest.mark.django_db
-    def test_get_user_by_nickname(self, generate_user):
+    def test_create_users(self, users):
+        # Test generarting users in DB
+        for i in range(len(users)):
+            assert users[i] in User.objects.all()
+
+    def test_get_user_by_nickname(self, user):
         # Tests that the user is in the DB
         user = User.get_user_by_nickname(NICKNAME)
-        assert generate_user.name == user.name
-        assert generate_user.nickname == user.nickname
-        assert generate_user.email == user.email
-        assert generate_user.password == user.password
+        assert user.name == user.name
+        assert user.nickname == user.nickname
+        assert user.email == user.email
+        assert user.password == user.password
```

Comment:
These assertions are always true, you compare an element to itself.

Reply:
@avi-biton Yes yes true I used generate_user before and after changing it to user I forgot to change the variable itself, updated.